quarkchain/cluster/miner.py

```diff
@@ -33,6 +33,7 @@ def validate_seal(
     block_header: Header,
     consensus_type: ConsensusType,
     adjusted_diff: int = None,  # for overriding
+    with_rotation_stats: bool = False,
```

Comment:
since `with_rotation_stats` only applies to qkchash and not others, I suggest using kwargs

```
def validate_seal(..., **kwargs):
  if consensus_type == POW_QKCHASH:
    if not qkchash_check_pow(..., kwargs.get("with_rotation_stats", False)): ...
```

can also change the naming to `qkchash_with_rotation_stats`